frontend/src/views/Lobby.tsx

```diff
@@ -46,6 +52,17 @@ function LobbyPage() {
     console.log(user);
   };
 
+  const handleClick = (): Promise<void> => {
+    const request = { roomId: currentRoomId, user: currentUser };
+    return axios.post<void>('/api/v1/start', request)
+      .then(() => {
+        setLoading(true);
+      })
+      .catch((err) => {
+        setError(err.message);
+      });
+  };
```

Comment:
nit: Move this post request to a new file (maybe `Game.ts`) in the `api` folder using a similar format as the requests in `Room.ts` (from the looks of it the new post request also returns a RoomDto)

Also, I think `handleClick` can be void if you decide to do this, but I may be wrong so lmk

Reply:
The Loading and Error states can't be accessed from outside this function I think. Is making a new file important? If so I can try to find a way around it.

Reply:
You should be able to do something like `callPostRequestImportedFromAnotherFile().then(...).catch(...)` as long as you make the request return a promise (if you look at `Room.ts` you can see what each function should look like as well as how those functions are used in other React files